python-sdk/langwatch/examples/langchain_rag_bot_vertex_ai.py

```
@@ -0,0 +1,116 @@
+        "langwatch_search",
+        "Search for information about LangWatch. For any questions about LangWatch, use this tool if you didn't already",
+    )
+
+    # Alternative approach to retrievers
+    # wrapped_tool = langwatch.langchain.capture_rag_from_tool(
+    #     retriever_tool, lambda response: [RAGChunk(content=response)]
+    # )
+
+    try:
+        credentials_json = json.loads(os.environ["GOOGLE_APPLICATION_CREDENTIALS"])
+        credentials_file = tempfile.NamedTemporaryFile(mode="w", delete=False)
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)